corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -120,20 +120,7 @@ def update_sync_log_with_checks(sync_log, xform, cases, case_db,
         sync_log.update_phone_lists(xform, cases)
     except SyncLogAssertionError, e:
```

Comment:
that makes me nervous given that i think it happens pretty frequently. maybe start with `soft_assert` to see if it's common?